package.json

```diff
@@ -4,6 +4,15 @@
   "description": "Seamless REST/GraphQL API mocking library for browser and Node.js.",
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
+  "exports": {
+    ".": "./lib/index.js",
+    "./iife": "./lib/iife/index.js",
```

Comment:
I don't think we promise `msw/iife` import. It's true in the case of `msw/node` and `msw/native` but as far as I reacal IIFE should be imported directly from the build directory like so:

```html
<script src="https://unpkg.com/msw/lib/iife/index.js"></script>
```

We don't have a placeholder directory `./iife` so I believe the `iife` export rule is misleading.

Reply:
You are right. I'll remove this.